### PR TITLE
Ohio 2016: Starting. Supporting "false" for TBD

### PR DESCRIPTION
**Pull request by kjenkins19**
This is the initial creation of the pages for Ohio 2016.
* Improving logic for testing if the Start Date, End Date, or CFP Date is not set (left blank) in the YML.
** I updated this since "isset" was being called after using the variable, making the existing check impossible to utilize.
* Created initial data and markup for Ohio 2016.

### CODE REVIEW

**Comment by mattstratton** [comment]
Can you pull out the "false" thing? I added a fix this morning that fits with some other uses. See #170

**Comment by mattstratton** [comment]
Let me rephase - can you pull down what I just pushed earlier today, merge it in with your changes if needed? I didn't get a chance to see everything you implemented your addition in Future works well.

File: content/events/2016-ohio/location.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++
+City = ""
+Year = ""
+date = "2016-03-06T21:17:00-06:00"
+title = "location"
+type = "event"
+draft = true
```
Comment (bridgetkromhout) [inline content/events/2016-ohio/location.md]: should be `draft = false`

File: content/events/2016-ohio/proposals.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
++++
```
Comment (bridgetkromhout) [inline content/events/2016-ohio/proposals.md]: You can remove this file if you like. We aren't going to support showing under-review proposals.

File: content/events/2016-ohio/welcome.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
++++
+date = "2016-03-06T21:15:25-06:00"
+title = "welcome"
+type = "event"
+City = "Ohio"
+Year = "2016"
+aliases = ["/events/2016-Ohio"]
```
Comment (bridgetkromhout) [inline content/events/2016-ohio/welcome.md]: The "Ohio" shouldn't be in caps here.

File: data/events/2016-ohio.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: "2016ohio" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
```
Comment (bridgetkromhout) [inline data/events/2016-ohio.yml]: We're going to "2016-ohio" for this.

File: data/events/2016-ohio.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+year: "2016" # The year of the event. Make sure it is in quotes.
+city: "Ohio" # The city name of the event. Capitalize it.
+friendly: "2016-Ohio" # Four digit year and the city name in lower-case. Don't forget the dash!
```
Comment (bridgetkromhout) [inline data/events/2016-ohio.yml]: Lowercase ohio.

File: data/events/2016-ohio.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+#navigationelements: ["welcome", "contact", "location", "program", "propose", "proposals", "registration", "conduct", "sponsor"] # List of pages you want to show up in the navigation of your page. Remove any that aren't relevent yet.
+navigationelements: ["welcome", "contact", "propose", "conduct", "sponsor"]
+team: ["Ben Blanquera", "Bill Schwanitz", "Dan Sajner", "David Laulusa", "Faye Thompson", "Kyle Jenkins", "Mark Goodmanson", "Michael Ducy", "Michael Swartzbaugh", "Rob Kinyon", "Warner Moore"]
+organizer_email: "organizers-ohio-2016@devopsdays.org" # Put your organizer email address here
+proposal_email: "organizers-ohio-2016@devopsdays.org" # Put your proposal email address here
+sponsors: # List all of your sponsors here along with what level of sponsorship they have.
+  - id: chef
+    level: gold
+  - id: datadog
+    level: gold
```
Comment (bridgetkromhout) [inline data/events/2016-ohio.yml]: Remove the chef and datadog sponsors if they haven't sponsored yet. (Sorry about that - bad example data.) You can leave ADO. :)

**Comment by bridgetkromhout** [comment]
I pulled this branch and it didn't build locally for me. Deferring to @mattstratton.

File: layouts/shortcodes/cfp_dates.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
 {{ $event_slug := index $path 1 }}
 {{ $e :=  (index .Page.Site.Data.events $event_slug) }}
 
-{{ if isset $e "cfp_date_start" }}
+{{ if ($e.cfp_date_start) }}
```
Comment (mattstratton) [inline layouts/shortcodes/cfp_dates.html]: Pull this out for now as it's not building for our tests - it works correctly on our test stuff. If you want to hack at it again, let's do a separate PR for it (vs the content you're adding)

File: layouts/shortcodes/event_end.html [hunk 8]
```diff
@@ -3,7 +3,7 @@
 {{ $e :=  (index .Page.Site.Data.events $event_slug) }}
 {{ $enddate := $e.enddate}}
 
-{{ if isset $e "enddate" }}
+{{ if $enddate }}
```
Comment (mattstratton) [inline layouts/shortcodes/event_end.html]: Pull this out for now as it's not building for our tests - it works correctly on our test stuff. If you want to hack at it again, let's do a separate PR for it (vs the content you're adding)

File: layouts/shortcodes/event_start.html [hunk 9]
```diff
@@ -4,8 +4,10 @@
 
 {{ $startdate := $e.startdate}}
 
-{{ if isset $e "startdate" }}
+{{ if $startdate }}
```
Comment (mattstratton) [inline layouts/shortcodes/event_start.html]: Pull this out for now as it's not building for our tests - it works correctly on our test stuff. If you want to hack at it again, let's do a separate PR for it (vs the content you're adding)
